ui/hooks/useTokensToSearch.js

```diff
@@ -83,7 +90,12 @@ export function useTokensToSearch({ usersTokens = [], topTokens = {} }) {
   const conversionRate = useSelector(getConversionRate);
   const currentCurrency = useSelector(getCurrentCurrency);
   const defaultSwapsToken = useSelector(getSwapsDefaultToken);
-
+  const tokenList = useSelector(getTokenList);
+  const useStatictokenList = useSelector(getUseStaticTokenList);
+  // TODO: replace contract-metadata with list from state and introduce shuffle to tokenlist
+  const shuffledTokenList = useStatictokenList
+    ? shuffledContractMap
+    : Object.values(tokenList);
```

Comment:
When I added shuffling to `Object.values(tokenList)` here, the application started crashing.